internal/service/project/resource_project.go

```diff
@@ -393,12 +394,21 @@ func GetProjectPropsFromAPI(ctx context.Context, projectsAPI admin.ProjectsApi,
 		return nil, fmt.Errorf("error getting project's slow operation thresholding enabled (%s): %v", projectID, err.Error())
 	}
 
+	var users []admin.GroupUserResponse
+	if isDataSource {
+		var err error
+		users, err = ListAllProjectUsers(ctx, projectID, mongoDBCloudUsersAPI)
+		if err != nil {
+			return nil, fmt.Errorf("error getting project's users (%s): %v", projectID, err.Error())
```

Comment:
nit: You can use `:=` short assignment statement to avoid the declaration of `err` with explicit type
```suggestion
		users, err := ListAllProjectUsers(ctx, projectID, mongoDBCloudUsersAPI)
		if err != nil {
			return nil, fmt.Errorf("error getting project's users (%s): %v", projectID, err.Error())
```

Reply:
Changed